ddpui/services/chart_service.py

```
@@ -0,0 +1,341 @@
+            List of dashboard info dictionaries
+        """
+        # Verify chart exists
+        ChartService.get_chart(chart_id, org)
+
+        # Find dashboards that have this chart in their components
+        dashboards_with_chart = []
+        dashboards = Dashboard.objects.filter(org=org)
+
+        for dashboard in dashboards:
+            if dashboard.components:
+                for component_id, component in dashboard.components.items():
```

Ruff (0.14.8)

Loop control variable `component_id` not used within loop body

Rename unused `component_id` to `_component_id`

(B007)